File: uzapijava/src/main/java/com/uiza/api/models/UZLiveCounter.java
package com.uiza.api.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
public class UZLiveCounter {
    @JsonProperty("views")
    private int views;

    public UZLiveCounter() {}

    public int getViews() {
        return views;
    }
}

File: uzapi/src/main/java/com/uiza/api/models/UZLiveCounter.kt
package com.uiza.api.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
data class UZLiveCounter constructor(@JsonProperty("views") var views: Int = 0) : Serializable
